Sources/AblyAssetTrackingInternal/AblyWrapper/SDK/AblyCocoaSDK.swift

```diff
@@ -104,10 +108,26 @@ struct AblyCocoaSDKRealtimePresence: AblySDKRealtimePresence {
 
 struct AblyCocoaSDKConnection: AblySDKConnection {
     fileprivate let connection: ARTConnection
+
+    func errorInfo() -> ErrorInformation {
+        guard let currentErrorReason = connection.errorReason else {
```

Comment:
Was done because the bits of code that needed it required it to be an ErrorInformation, so have shifted that default value logic to the call site in c152e37